### DISCUSSION TITLE
Is "feature hashtags" capability still available in mastodon?

### DISCUSSION THREAD

**Discussion by StavrosKalognom** [mastodon/mastodon#27781] [Q&A] [answered]
I am wondering if the feature "feature hashtags" still works in mastodon. I can see the feature hashtags that I had set in settings->profile->#feature hashtags, but they do not appear in the main page of my profile (mastodon.social@username) neither in mastodon.social/deck/@username, in order to choose one of them, (I mostly use mozila or chrome).

**Comment by vmstan** [comment]
Yes

<img width="338" alt="image" src="https://github.com/mastodon/mastodon/assets/3002053/3c5a654f-3d63-4eb1-8e13-b3ae2f812444">

**Reply by StavrosKalognom** [reply]
Unfortunately, it does not appear to my profile what you display in the picture, as it used to do so in the past (I do not know if I had done something wrong in settings). However, I saw in a tutorial video that the pattern for the url concerning featured hashtags is (e.g. for tag #music) https://mastodon.social/@myprofilename/tagged/music. When I enter this url (with my profile name, instead of myprofilename) it shows all of my toots that have the tag #music. I am thinking about pinning a toot that has all of my featured hashtags, as an alternative solution.

**Reply by StrangeMinds1960** [reply]
I've just started playing with featured hashtags, they do not appear under https://strangeminds.social/deck/@danny, but do appear under https://strangeminds.social/@danny

You haven't supplied your Mastodon username, so only you can say if this is the cause

**Reply by StavrosKalognom** [reply, marked as answer]
Unfortunately, it does not appear neither under https://mastodon.social/@oregano so I decided to pin the following post: 

"If you want to see all of my toots that have a specific hashtag, you can just enter in your browser the url: mastodon.social/@oregano/tagged/tagname and replace the tagname with the specific hashtag (e.g for #music, type mastodon.social/@oregano/tagged/music). I often use the hashtags #learning, #petition, #music, #donate and occasionally the hashtags #news, #engineering, #traditionalmusic, #math. I hope that you liked this tip 🙂."

**Reply by StrangeMinds1960** [reply]
Yes it does, perhaps if you log out you might see it (p.s. not sure how the screenshot I've added will show up)

![hashtags](https://github.com/mastodon/mastodon/assets/121610247/83c26c7f-2981-46bd-b18b-3533ab0ba709)

**Reply by StavrosKalognom** [reply]
That is really strange. The image below displays what it is appeared at my laptop screen. Even when I search in the page https://mastodon.social/@oregano for the words FEATURED HASHTAGS nothing appears. I do not know if is a weird setting of my browser (chrome). 

![image](https://github.com/mastodon/mastodon/assets/36454014/3a5760e1-d72b-4943-b1ae-5dee4290f667)

Anyways, with the post that I pinned, everyone can search for any hashtag that I have used (not just the three most used). By the way, I wish a happy new year to all of you :) .
